### PR TITLE
PPF-167-navigation

### PR DESCRIPTION
**Pull request by vhande**
### Added
- HeaderMobile
- NavigationElements
- SecondaryButton

Desktop:
HeaderDesktop
Mobile:
HeaderMobile

localhost_(iPhone SE)

---
Ticket: https://jira.uitdatabank.be/browse/...

### CODE REVIEW

File: resources/ts/Shared/NavigationElements.tsx [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+      <Link href="/">
+        <Heading
+          className="max-md:text-2xl py-2 border-transparent border-b-4"
+          level={3}
+        >
+          {t("title")}
+        </Heading>
+      </Link>
+      <div className="flex max-md:flex-col md:gap-8 min-w-[50%]">
+        {["integrations", "support"].map((pageTitle) => (
```
Comment (brampauwelyn) [inline resources/ts/Shared/NavigationElements.tsx:77]: Maybe you can define the pages in a `const` above? 
```javascript
const pages = ["integrations", "support"]
```

File: resources/ts/Shared/SecondaryButton.tsx [hunk 2]
```diff
@@ -1,18 +1,24 @@
...
 
-type Props = InertiaLinkProps;
+import { classNames } from "../utils/classNames";
 
-export const SecondaryButton = ({ children, ...props }: Props) => {
+type Props = {
+  orientation?: string;
+} & ComponentProps<"button">;
+
+export const SecondaryButton = ({ children, className, ...props }: Props) => {
```
Comment (brampauwelyn) [inline resources/ts/Shared/SecondaryButton.tsx:9]: I would prefer to invert the naming (both in the filename as in the variable name).

```suggestion
export const ButtonSecondary = ({ children, className, ...props }: Props) => {
```

File: resources/ts/Shared/SecondaryLinkButton.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { Link, InertiaLinkProps } from "@inertiajs/react";
+import { classNames } from "../utils/classNames";
+
+type Props = InertiaLinkProps;
+
+export const SecondaryLinkButton = ({
```
Comment (brampauwelyn) [inline resources/ts/Shared/SecondaryLinkButton.tsx:7]: Same here I prefer to invert the naming. 
So you only need to read the first part of the file to know that it's about a button...

```suggestion
export const ButtonLinkSecondary = ({
```

**Review by simon-debruijn — CHANGES_REQUESTED** [review]
In mobile when we show the modal/dialog, we should maybe show a darker backdrop behind it. Now the modal doesn't stand out enough compared to the content behind it.

**Review by simon-debruijn — APPROVED** [review]

**Pull request merged**